swagger.yml

```diff
@@ -47,7 +47,7 @@ paths:
       - default
 info:
   title: AIOPS Data Collector API
-  version: 0.0.1
+  version: 0.1
```

Comment:
Noticed something really strange here...

If you copy-paste swagger.yml in http://editor.swagger.io/, there is an error thrown for this line.
Apparently, it is expecting -
```yaml
version: '0.1'
```
For some reason it did not mind that the original version value (0.0.1) was not a string -- strange!

Anyway, can you change it to `'0.1'`?

Reply:
Nice YAML parsing conflict. :smile: If it can't convert the value to a number, etc., it defaults to a string when YAML is parsed. So `0.0.1` would be treated as a string. And Swagger seems to enforce version to be string. So everything is fine there. Once I've changed it to `0.1`, this value can be converted and parsed as a number. Therefore number it is. But version is required to be a string and boom, error is born. :smile: Fun. I'm fixing it now. :+1:

Reply:
Alright, thanks for that explanation! :)

Reply:
Thanks for catching this! I wouldn't notice otherwise... :+1: